app/controllers/internal/configs_controller.rb

```diff
@@ -28,32 +28,24 @@ def create
 
   def config_params
     allowed_params = %i[
-      favicon_url
       ga_view_id ga_fetch_rate
-      logo_png
-      logo_svg
-      main_social_image
-      mascot_image_description
-      mascot_image_url
-      mascot_user_id
-      onboarding_taskcard_image
       periodic_email_digest_max
       periodic_email_digest_min
-      primary_sticker_image_url
-      shop_url
       sidebar_tags
-      suggested_tags
       twitter_hashtag
-      suggested_users
-      tagline
+      shop_url
       payment_pointer
+      health_check_token
```

Comment:
health_check_token field was not updating correctly as it wasn't added to the allowed params.